eng/pipelines/spec-gen-sdk.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  - name: SpecRepoCommit
+    type: string
+    default: 'HEAD'
```

Comment:
Should these be branch names like main by default?

Reply:
Perhaps, but does it make any difference?

Reply:
It might not make a difference but it depends on how the repo was cloned. For example, if this was from a PR it will HEAD will be the head of the PR branch not main. So, we will need to understand the usage patterns to determine what will be the best option.